mobile/src/navigations/root/root.tsx

```diff
@@ -1,24 +1,56 @@
+import {
+  type BottomTabNavigationOptions,
+  createBottomTabNavigator,
+} from '@react-navigation/bottom-tabs';
 import {
   createNativeStackNavigator,
   type NativeStackNavigationOptions,
 } from '@react-navigation/native-stack';
 import React from 'react';
 
-import { RootScreenName } from '#libs/enums/enums';
-import { type RootNavigationParameterList } from '#libs/types/types';
+import { HomeIcon } from '#assets/img/icons/icons';
+import { AppColor, RootScreenName } from '#libs/enums/enums';
+import {
+  type RootNavigationParameterList,
+  type TabNavigationParameterList,
+} from '#libs/types/types';
 import { Auth } from '#screens/auth/auth';
+import { Home } from '#screens/main/home';
+
+import { styles } from './styles';
 
 const NativeStack = createNativeStackNavigator<RootNavigationParameterList>();
+const BottomTab = createBottomTabNavigator<TabNavigationParameterList>();
 
 const screenOptions: NativeStackNavigationOptions = {
   headerShown: false,
 };
 
+const tabNavigatorOptions: BottomTabNavigationOptions = {
+  tabBarActiveTintColor: AppColor.BLUE_200,
+  tabBarIcon: HomeIcon,
+  tabBarInactiveTintColor: AppColor.GRAY_300,
+  tabBarShowLabel: false,
+  tabBarStyle: styles.tabBarStyle,
+};
+
+const BottomTabsNavigator: React.FC = () => {
```

Comment:
I suggest we don't create all navigations in one file. Let's create a separate file for bottom tab navigation

Reply:
Done, but I am not sure that I did it in the right way. Particularly, I am not sure about the naming conception - Main Stack, which has only TabNavigationBottom stack. Can it be done this way?

Reply:
Not really sure about MainStack here. At least because we are not using Stack navigator here. Maybe just Main?